distribution.md

```diff
@@ -69,3 +70,38 @@ Each stack ID MUST only be present once.
 For a given stack, the `mixins` list MUST enumerate mixins such that no included buildpacks are missing a mixin for the stack.
 
 Fewer stack entries as well as additional mixins for a stack entry MAY be specified.
+
+### Build Image
+
+The following defines how a `build-image` SHOULD be packaged for distribution as an OCI Image. The `build-image` is the component that provides the base image from which the build environment is constructed.
```

Comment:
Since the build image is not an executable, I would suggest that we don't "pre format" it.

```suggestion
The following defines how a build image SHOULD be packaged for distribution as an OCI Image. The build image is the component that provides the base image from which the build environment is constructed.
```